addon/mixins/axis-titles.js

```diff
@@ -192,7 +224,7 @@ const AxisTitlesMixin = Ember.Mixin.create({
   updateYAxisTitle: function(){
     this.get('yAxisTitle')
     .text(this.get('yAxisTitleDisplayValue'))
-    .style('text-anchor', 'start').attr({
+    .style('text-anchor', 'middle').attr({
```

Comment:
The title is supposed to be centered. The x title is actually `middle`. I think putting `start` here is a mistake.